cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -193,6 +193,10 @@ private static void warningsForRemovedEndVars(Map<String, String> map) {
             LOGGER.warn("Kafka TLS sidecar container has been removed and the environment variable {} is not used anymore. " +
                     "You can remove it from the Strimzi Cluster Operator deployment.", STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE);
         }
+        if (map.containsKey(STRIMZI_DEFAULT_TLS_SIDECAR_CRUISE_CONTROL_IMAGE))    {
+            LOGGER.warn("Cruise Control TLS sidecar container has been removed and the environment variable {} is not used anymore. " +
+                    "You can remove it from the Strimzi Cluster Operator deployment.", STRIMZI_DEFAULT_TLS_SIDECAR_CRUISE_CONTROL_IMAGE);
+        }
```

Comment:
I wonder if this is really worth it. Also, how long are we going to keep this? The Kafka sidecar was removed quite some time ago.

Reply:
I agree we could remove the one for the Kafka sidecar maybe but at this point we should keep the CC one because it's a new removal.